### PR TITLE
A bunch of random item reworks: 11

### PR DESCRIPTION
**Pull request by Karol1223**
#### Summary
None

#### Describe the solution

<details>
<summary>BODY BAG</summary>

- Weight: 1500g -> 1061g
- Volume: 1l -> 5.5l
- Longest side: removed. It is a soft item that would not really apply to manually defined longest side
- ``price_preapoc``: 0$ -> 30$
- Initial implementation of a disassembly recipe
- ``to-hit`` values now use modifiers
- Added ``NO_SALVAGE`` flag, because you shouldn't be able to cut up a soft bag to get solid chunks

</details>

<details>
<summary>PIZZA CUTTER</summary>

- Weight: 190g -> 186g
- Volume: 135ml -> 350ml
- ``price_preapoc``: 1$ -> 6$
- ``to-hit`` values now use modifiers

</details>

<details>
<summary>POOL CUE</summary>

- Weight: 520g -> 510g
- Volume: 2l -> 1042ml
- Longest side: 150cm -> 148cm
- ``price_preapoc``: 80$ -> 15$
~~- Bash damage: 10 -> 7 - recalculated proportionally from crowbar's weight and bash damage, since it felt too damaging~~ I can't do that because it has a DPS test. I still maintain that how strong it is is bullshit.
- Added ``FRAGILE_MELEE`` flag, because slapping people with a pool cue means it's not gonna have a long lifespan

</details>

<details>
<summary>FLATJAW TONGS</summary>

- Weight: 750g -> 730g
- Volume: 500ml -> 662ml
- Longest side: 30cm -> 44cm
- ``price_preapoc``: 16$ -> 39$
- Bash damage: 6 -> 9 - recalculated proportionally from crowbar's weight and bash damage, since I didn't like that it weighed more than the pool cue but did less damage on hit (still does now because of pool cue favorization)
- Initial implementation of a disassembly recipe
- ``to-hit`` values now use modifiers

</details>

<details>
<summary>BINOCULARS</summary>

- Category: ``clothing`` -> ``tools``
- Weight: 500g -> 550g
- Volume: 500ml -> 540ml
- Longest side: N/A -> 15cm
- ``price_preapoc``: 300$ -> 34$
- Edited the disassembly recipe to need a cutting tool but provide sufficiently more plastic
- ``to-hit`` values now use modifiers
- Added proportions to material, so that it's 9:1 plastic and glass, instead of 1:1

</details>

Also fixed the boltcutters disassembly while I'm here because apparently I fucked up the syntax the last time. I still maintain we need a test to catch that.

#### Additional context
I did not expect to find body bags for sale on Amazon

### CODE REVIEW

File: data/json/items/melee/bludgeons.json [hunk 1]
```diff
@@ -1442,18 +1442,21 @@
     "id": "pool_cue",
     "name": { "str": "pool cue" },
     "description": "A hardwood stick designed for hitting colorful balls around a felt table.  Truly, the coolest of sports.",
-    "weight": "520 g",
+    "weight": "510 g",
+    "volume": "1042 ml",
+    "//1": "This assumes the pool cue is a cylinder with the length from the amazon listing and the diameter of 30mm as google said it is a likely diameter if the pin is 13mm.",
```
Comment (TheShadowFerret) [inline data/json/items/melee/bludgeons.json:1447]:
```suggestion
    "//1": "This assumes the pool cue is a cylinder with the length from the Amazon listing and the diameter of 30mm as Google said it is a likely diameter if the pin is 13mm.",
```

Comment (TheShadowFerret) [inline data/json/items/melee/bludgeons.json:1447]:
```suggestion
    "//1": "This assumes the pool cue is a cylinder with the length from the Amazon listing and the diameter of 30mm as Google said it is a likely diameter if the pin is 13mm.",
```

**Review by TheShadowFerret — COMMENTED** [review]
>I did not expect to find [insert item] for sale on Amazon

I don't believe you 🤔

**Comment by Karol1223** [comment]
> I don't believe you 🤔

Amazon very adamantly refused to show me results for "butcher knife" (only showed me one set of knives and the rest was knife blocks/holders. Then it very adamantly showed me a lot of results for "cadaver bags", some of which had imagery which I would only describe as things I wouldn't share around a dinner table (not graphic, but corpses/fake corpses in body bags isn't that fun either)

**Pull request merged**